odoo-shippable/scripts/build-image.sh

```diff
@@ -71,6 +75,13 @@ apt-get install ${DPKG_DEPENDS} ${PIP_DPKG_BUILD_DEPENDS}
 # Install node dependencies
 npm install ${NPM_OPTS} ${NPM_DEPENDS}
 
+# Install virtualenv for each version of python
+for version in '3.3' '3.4' '3.5'
```

Comment:
@moylop260 
I try to run `python3.6 -m pip install virtualenv`
![image](https://cloud.githubusercontent.com/assets/1387970/26059653/841faa62-3950-11e7-9a3c-b1a6232038ba.png)